src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -501,6 +524,7 @@ public void WriteTo(ObjectWriter writer)
                 writer.WriteString(FilePath);
                 writer.WriteString(OutputFilePath);
                 writer.WriteString(OutputRefFilePath);
+                writer.WriteString(DefaultNamespace);
                 writer.WriteBoolean(IsSubmission);
                 writer.WriteBoolean(HasAllInformation);
```

Comment:
@heejaechang is this information ever persisted anywhere? is changing the format version a problem?  does some format-version-value need to be updated here so nothing in our system uses the previous format?

Reply:
As far as I know, this is only OOP over-the-wire serialization and so there isn't a version to update.

Reply:
no one currently persist this data. when we do, we probably want to have format option but also need to decide what that means for OOP. since when format is not same, there is nothing it can do to recover.